index.html

```diff
@@ -53,12 +57,11 @@ <h1>PokeYou!</h1>
     <div class="container" id="screentwo">
         <div class="row">
             <!-- Image input card -->
-            <div class="card col-4" id="User-Image-Container" style="width: 18rem;">
+            <div class="card col-4 cardOne" data-aos="flip-down" id="User-Image-Container" style="width: 18rem;">
                 <div class="card-body">
                     <h5 class="card-title">Upload your photo below:</h5>
                     <h6 class="card-subtitle mb-2 text-muted">Card subtitle</h6>
-                    <p class="card-text">Some quick example text to build on the card title and make up the bulk of the
-                        card's content.</p>
+                    <p class="card-text">Have the picture render here.</p>
```

Comment:
Add an ID so we can just update this HTML element for adding the picture. It shares a class with the element above it.